index.html

```diff
@@ -121,6 +121,33 @@ <h2>A project for first-time contributions</h2>
 
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Brynjar H</H1></p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com/BinnihH" target="_blank">BinnihH</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com/binnih" target="_blank">binnih</a>
+          </p>
+          <p class="about">Learning to contribute to open source.</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="#" target="_blank" title="First Resource">Resource 1</a>
```

Comment:
You also need to change
1.  the href tag in every <a>
2. "Resource X" text
3. title in every <a>

such as
```suggestion
                <a href="https://www.theodinproject.com/" target="_blank" title="The Odin Project">The Odin Project</a>
```